lib/index.js

```diff
@@ -22,6 +22,7 @@ async function run() {
   try {
     const gitHubKey = process.env.GITHUB_TOKEN || core.getInput('github_token', { required: true });
     const filesToIgnore = core.getInput('files_to_ignore', { required: true });
+    const filesToInclude = core.getInput('files_to_include', { required: false });
```

Comment:
```suggestion
    const filesToInclude = core.getInput('files_to_include');
```

I don't think we need it, it will only confuse in future why we have it here but not for other inputs